docs/xshareclasses.md

```diff
@@ -384,6 +384,14 @@ behavior, which can improve the performance of class loading from the shared cla
 
 :   <i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** On macOS systems, you must set `kern.sysv.shmmax` and `kern.sysv.shmall` when using a nonpersistent cache.
 
+### `noPersistentDiskSpaceCheck`
+
+        -Xshareclasses:noPersistentDiskSpaceCheck
+
+:   Instructs the VM not to check for available storage on the file system before creating a persistent shared classes cache. This option prevents an unnecessary step for file systems that do not support the checking of free space. Regardless of whether  you choose to set this option, if there isn't enough disk space available when the VM writes to the shared cache memory, a **SIGBUG** or **SIGSEGV** signal occurs and the VM ends. 
```

Comment:
Not sure what `This option prevents an unnecessary step` means. The problem was that on a file system which didn't support free space checking, the free space came back as zero, preventing the shared cache from being created.

Reply:
Thanks. I didn't have time to root through the underlying issue. Reasoning updated.